Lib/test/test_cppext.py

```diff
@@ -12,7 +16,7 @@
 MS_WINDOWS = (sys.platform == 'win32')
 
 
-SETUP_TESTCPPEXT = support.findfile('setup_testcppext.py')
+PKG_CPPEXTDATA = os.path.dirname(support.findfile('setup.py', subdir="cppextdata"))
```

Comment:
This is the unadorned directory name, which is treated below by pip as a *package* name, not a directory.

Add an `os.path.abspath()` round this to disambiguate?